documentation/IDTA-01005/modules/ROOT/pages/changelog.adoc

```diff
@@ -15,11 +15,10 @@ SPDX-License-Identifier: CC-BY-4.0
 This annex lists the changes from version to version of the file exchange format together with major changes in the overall document. Non-backward compatible changes (nc) are marked as such.
 
 
-== Changes V3.1 vs. V3.0
+== Changes V3.1 vs. V3.0.1
```

Comment:
If this document is released as V3.0.1, a reference to V3.1 (i.e., the future release coming after V3.0.1) would be rather confusing.

Reply:
It will not, V3.0.1 will be published as .pdf as V3.0. This is version V3.1 but of course the changes of V3.0.1 are incorporated as well.